src/main/java/com/commercetools/sdk/jvm/taglets/DocumentationTaglet.java

```diff
@@ -127,7 +123,8 @@ private String getString(final Tag tag) throws IOException {
             updateActionNames.forEach(name -> builder.append(format("<li><a href=\"%s/%s.html\">%s</a></li>", UPDATEACTIONS_PACKAGE, name, name)));
             builder.append("</ul>");
             result = builder.toString();
-        } else if (isClientRequestList(tag)) {
+        }
+        else if (isClientRequestList(tagText)) {
```

Comment:
Here is the indentation odd, I wouldexpect `} else if (isClientRequestList(tagText)) {`